src/App.js

```diff
@@ -1,9 +1,33 @@
-function App () {
+import { useState } from "react";
+import birdData from "./data/birds";
+import Bird from "./components/Bird";
+import "./App.css";
+import Cart from "./components/Cart";
+import Checkout from "./components/Checkout";
+
+function App() {
+  const [cart, setCart] = useState([]);
+  const addToCart = (bird) => {
+    bird = { ...bird, id: Math.random() };
+    setCart((prev) => [...prev, bird]);
+  };
+
   return (
-    <div>
-      <h1>Hello, world!</h1>
+    <div className="app">
+      <h1>Bird Sanctuary Project</h1>
+      <div className="grid">
+        <div className="side-bar">
+          <Cart cart={cart} setCart={setCart} />
+          <Checkout setCart={setCart} />
+        </div>
+        <div className="cards">
+          {birdData.map((bird) => (
+            <Bird bird={bird} key={bird.id} addToCart={addToCart} />
+          ))}
+        </div>
```

Comment:
I like the solution of giving each bird a unique id and it's something we're going to be seeing more when we start using databases. Using the index from the map you could implement a simpler, more efficient deleting method for the birds in the cart. Can you see how?